nemo/collections/nlp/modules/common/megatron/megatron_utils.py

```diff
@@ -136,12 +137,14 @@ def get_megatron_checkpoint(pretrained_model_name):
         path (str): path to model checkpoint
     '''
     url = MEGATRON_CONFIG_MAP[pretrained_model_name]['checkpoint']
-    path = os.path.join(MEGATRON_CACHE, pretrained_model_name)
+    cache_dir = os.path.join(MEGATRON_CACHE, pretrained_model_name)
+    path = os.path.join(cache_dir, 'model_optim_rng.pt')
 
```

Comment:
filename defined already contains .pt. For non model-parallel checkpoint, we're asking users to provide path to the file not dir

Reply:
that wasn't the case for me, path was just the directory. pretrained_model_name is just "megatron-bert-345m-uncased"

Reply:
"megatron-bert-345m-uncased" is supposed to be a filename, if you use 'model_optim_rng.pt' cased and uncased models will be confused

Reply:
does torch/megatron dir exist? @ericharper

Reply:
oh okay, should we put a .pt or something on it? To make that more clear?

Reply:
MEGATRON_CACHE doesn't necessarily exist.